### DISCUSSION TITLE
Right click in Screen

### DISCUSSION THREAD

**Discussion by hervehobbes** [adirh3/Fluent-Search#1074] [Q&A] [answered]
Hello,
I'm using "Search in-app using Screen hotkey (Ctrl + M)" but I don't understand how to do a right click or a "Double click gesture" for example. Once I typed Ctrl + M, I can see the two letters on clickable elements but how to simulate a right click on one of them?

**Comment by adirh3** [comment, marked as answer]
To use mouse gestures with Screen Search do the following -
1. Invoke screen search via hotkey (default Ctrl+M)
2. Press the key `2` for double click, or `4 ` for right click
3. Type the hint you want to click on

**Comment by hervehobbes** [comment]
Thank you very much!
